wandb/stats.py

```diff
@@ -151,28 +158,30 @@ def stats(self):
             handle = nvmlDeviceGetHandleByIndex(i)
             try:
                 util = nvmlDeviceGetUtilizationRates(handle)
+                memory = nvmlDeviceGetMemoryInfo(handle)
                 temp = nvmlDeviceGetTemperature(handle, NVML_TEMPERATURE_GPU)
                 stats["gpu.{0}.{1}".format(i, "gpu")] = util.gpu
-                stats["gpu.{0}.{1}".format(i, "memory")] = util.memory
+                stats["gpu.{0}.{1}".format(i, "memory")] = memory.used
```

Comment:
I think we might need to use a different key unfortunately. The UI needs to be able to display the correct label which depends on which CLI version sent it.

An alternative would be to store the CLI version on each run in the DB, then the UI could switch on that.